samplers/jaegerremote/sampler_remote_options.go

```diff
@@ -48,6 +54,7 @@ func newConfig(options ...Option) config {
 			MaxOperations:            defaultSamplingMaxOperations,
 			OperationNameLateBinding: defaultSamplingOperationNameLateBinding,
 		},
+		logger: stdr.NewWithOptions(log.New(os.Stderr, "", log.LstdFlags|log.Lshortfile), stdr.Options{LogCaller: stdr.Error}),
```

Comment:
Use a no-op logger here.  It is bad form for libraries to be emitting output to STDOUT.  The user can provide a logger using the new Option.

Reply:
I think it should output to STDOUT. (aws/xray is the same)
If the user does not configure this option this error will not be visible.
I think most users will use the default option.

Reply:
Agreed, please use a no-op logger here. Debugging will require a positive action to enable logging, the instrumentation should not spew output to a console by default. Especially since that is not the current behavior.